exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -314,6 +370,31 @@ public void deleteAllOptions(OptionType type) {
     }
   }
 
+  public void populateDefualtValues(Map<String, OptionValidator> VALIDATORS) {
+
+    // populate the options from the config
+    final Map<String, OptionValidator> tmp = new HashMap<>();
+    for (final Map.Entry<String, OptionValidator> entry : VALIDATORS.entrySet()) {
+
+      OptionValidator validator = entry.getValue();
+      final OptionValue.Kind kind = validator.getKind();
+      String name = entry.getValue().getOptionName();
+      OptionValue value;
+      String configPath = "drill.exec.options.";
+      String configName = configPath + name;
+      try {
+        value = validator.loadConfigDefault(bootConfig,name,configPath);
+        validator.setDefaultValue(value);
```

Comment:
Setting the default should be done in `loadConfigDefault`. Actually, since there is only one place to load the default from, the name `loadDefault` is fine.